module/VuFind/src/VuFind/ILS/Driver/Demo.php

```diff
@@ -472,6 +472,7 @@ protected function createRequestList($requestType)
                 "expire"   => $this->dateConverter->convertToDisplayDate(
                     'U', strtotime("now + 30 days")
                 ),
+                "reqnum"   => sprintf("%06d", $i),
```

Comment:
@EreMaijala, this line got removed during static analysis because there's a duplicate "reqnum" property two lines further down. If we need this version for things to work, can we get rid of the other one?

Reply:
Oops. That was accidentally added when I merged the changes from an old working copy. Now removed.